Makefile

```diff
@@ -11,23 +11,22 @@ help:
 
 builds:
 	@echo "Building Docker Images... 🐳"
-	docker-compose build --no-cache
+	docker-compose build
 	@echo "Done! ✨"
-up:
+up: builds
 	@echo "Starting library... ⚡️ "
-	docker-compose up --build
+	docker-compose up --build -d
 	@docker-compose ps
 	@echo "Done! ✨"
 	@echo "Storybook Library is running on http://localhost:6006 🌐"
 	open http://localhost:6006
+	@docker-compose logs --follow
 stop:
 	@echo "Stopping Library 🛑"
 	docker-compose down
 	@echo "Done! ✨"
 	@echo "Sucessfully stopped Library"
-all:
-	@echo "Making library..."
-	make builds && make up
+all: builds up
```

Comment:
Adding `builds` and `up` as prerequisites will tell `make` to run fulfill those targets beforehand, so you don't need to run `make builds && make up` within the rule.

Reply:
wow thats super helpful! thanks! :)